pkg/manifests/types.go

```diff
@@ -601,6 +601,9 @@ type AlertmanagerUserWorkloadConfig struct {
 // The `PrometheusRestrictedConfig` resource defines the settings for the
 // Prometheus component that monitors user-defined projects.
 type PrometheusRestrictedConfig struct {
+	// Configures the default interval between consecutive scrapes in case the ServiceMonitor or PodMonitor resource does not specify any value.
```

Comment:
```suggestion
	// Configures the default interval between consecutive scrapes in case the ServiceMonitor or PodMonitor resource does not specify any value.
	// The interval needs to be greater than or equal to 5 seconds and less than or equal to 5 minutes. 
	// The value can be expressed as:
	// * seconds, for example `30s`.
	// * minutees, for example `1m`.
	// * a mix of minutes and seconds, for example `1m30s`.
```